presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -99,6 +99,12 @@ Mathematical Functions
     The lambda parameter must be a positive real number (of type DOUBLE).
     The probability p must lie on the interval [0, 1).
 
+.. function:: inverse_weibull_cdf(a, b, p) -> double
+
+    Compute the inverse of the Weibull cdf with given parameters a, b for the probability p.
```

Comment:
I won't be too picky, but either back-quote all variables, or none. You are doing it for the `weibull_cdf` but not here. Also, no need to have a new line for every sentence.